server/container/guice/rabbitmq/src/main/java/org/apache/james/modules/rabbitmq/RabbitMQModule.java

```diff
@@ -40,6 +61,40 @@
 
     @Override
     protected void configure() {
+        bind(Clock.class).toInstance(Clock.systemUTC());
+        bind(ThreadLocalRandom.class).toInstance(ThreadLocalRandom.current());
+        bind(CassandraMailQueueViewConfiguration.class).toInstance(CassandraMailQueueViewConfiguration
```

Comment:
We need a another properties file for `CassandraMailQueueViewConfiguration`?

Reply:
For a first implementation, it's normal to have some values _fixed_.